src/ResourceManagement/Automation/AutomationManagement/Microsoft.Azure.Management.Automation.nuget.proj

```diff
@@ -5,7 +5,7 @@
     Microsoft.Azure.Management.Automation
     -->
     <SdkNuGetPackage Include="Microsoft.Azure.Management.Automation">
-      <PackageVersion>2.0.2</PackageVersion>
+      <PackageVersion>2.0.4</PackageVersion>
```

Comment:
@vrdmr @safeermohammed the only option is to see the nuget.org release date and track it by finding the publish jobs that were run around that time.
We are trying to put in more meta data that will allow us to track this much better in the future.